### ISSUE TITLE
Unable to select BarChartView from storyboard (iOS)

### ISSUE THREAD

**Issue by Domsware** [danielgindi/Charts#1892]
Same problem as https://github.com/danielgindi/Charts/issues/1197 but for iOS.

I want to integrate Charts to my Swift 2.3 project, Xcode 8
Installation using Carthage

And I can't set BarChartView on graph view on StoryBoard.

**Comment by petester42** [comment]
You might have to build charts once first for things to show up.

**Comment by Domsware** [comment]
@petester42 

I don't understand your answer as Charts is installed using Carthage.

**Comment by paulpilone** [comment]
@petester42 Make sure you also set the Module in Interface Builder (below the "Class" setting) to "Charts".

**Comment by petester42** [comment]
What @paulpilone said as well.

**Comment by Domsware** [comment]
@paulpilone I already tried to set class settings on StoryBoard:

- I dragged & Dropped a view on Controller
- I then want to set Class Info on Identity Inspector: whether I use autocompletion or selection `BarChartView` is not offered as a choice
- manually adding `BarChartView` to **Class** field of **Custom Class** tab lead to a crash when accessing the view
- manually adding **Charts** to **Module** field of **Custom Class** tab does not work as Xcode remove it instantly

Currently I create a `View` on StoryBoard and get its configuration on a `BarChartView` I programmatically create.

**Charts** has been installed using classic Carthage command `carthage update --platform ios`.

**Comment by mossby** [comment]
@Domsware I was having same issue as you. Seems the trick is in Interface Builder set the Class first ("BarChartView") and then set the Module ("Charts").  If you set the module first Xcode will auto delete it as you mentioned. You have to manually type in the Module name but it doesn't delete itself if the Class is already set.

**Comment by Domsware** [comment]
@mossby Thank you for you help. If my memory is correct I've tried to do this but without success.

**Comment by mossby** [comment]
@Domsware I just found out you can also just subclass (i.e. MyBarChart: BarChartView { }) and then you don't have to set the module at all.

**Comment by liuxuan30** [comment]
I think this can be closed
